app/serializers/api/declaration_serializer.rb

```diff
@@ -0,0 +1,44 @@
+module API
+  class DeclarationSerializer < Blueprinter::Base
+    identifier :ecf_id, name: :id
+    field(:type) { "participant-declaration" }
+
+    class AttributesSerializer < Blueprinter::Base
+      exclude :id
+
+      field(:participant_id) { |declaration| declaration.user.ecf_id }
+      field(:declaration_type)
+      field(:course_identifier)
+      field(:declaration_date)
+      field(:updated_at)
+      field(:state) { |declaration| declaration.state.dasherize }
+      # TODO: implement once we have outcomes
+      field(:has_passed) { nil }
+
+      view :v1 do
+        field(:voided_state?, name: :voided)
```

Comment:
Should we remove these 2 as they're deprecated in favour of `state`?

https://manage-training-for-early-career-teachers.education.gov.uk/api-reference/reference-v1.html#schema-participantdeclarationattributes

Reply:
One for @cwrw but I expect we'll need to keep it in v1 as it would be a breaking change to remove it

Reply:
yea I think let's keep it as otherwise will be a schema change